examples/train_model.py

```diff
@@ -26,7 +26,7 @@
 
 # Constants
 features = FEATURES.DEEPCORE
```

Comment:
Given that this file does not contain any substantial changes that relate to the topic of this PR, I would revert its changes (such that it doesn't show up under "Changed files").